src/components/WalletModal/index.tsx

```diff
@@ -194,32 +199,27 @@ export default function WalletModal({
     setPendingWallet(connector) // set wallet for pending view
     setWalletView(WALLET_VIEWS.PENDING)
 
-    // if the connector is walletconnect and the user has already tried to connect, manually reset the connector
-    if (connector instanceof WalletConnectConnector) {
-      connector.walletConnectProvider = undefined
+    //     // if the connector is walletconnect and the user has already tried to connect, manually reset the connector
+    //     if (connector instanceof WalletConnect) {
+    //       connector.walletConnectProvider = undefined
+    //     }
+
+    if (connector) {
+      try {
+        connector.activate()
+        logMonitoringEvent({ account })
```

Comment:
unclear if `connector.getAccount` is same as `account` in v8

Reply:
hm bit weird that we used `getAccount`, probably shouldn't be that way haha - in v8 you can't query connectors for the account(s) - gotta get it from the store